src/app/concern/file-uploader/file-uploader.component.ts

```diff
@@ -24,27 +25,36 @@ export class FileUploaderComponent implements OnInit {
     "text/plain",
     "text/csv"
   ];
+  public acceptedFileSize = 1048576;
```

Comment:
i think you should speak to ify about a max file size limit as the requirement in the story is to not have a file size restriction...

Reply:
Absolutely after chatting on the group: I think Ify is fine with the solution, as she re-worded the text during conversations to 10MB

Reply:
sounds good